core/backend/src/internal/workspace/WorkspaceImpl.ts

```diff
@@ -229,6 +229,9 @@ class WorkspaceDbImpl implements WorkspaceDb {
     }
   }
   public get version() {
+    const cloudContainer = this.container.cloudContainer;
+    if (undefined === cloudContainer)
+      return "1.0.0"; // local file, no versioning. return default
```

Comment:
Why 1.0.0 instead of 0.0.0?

Reply:
local files dont have versions, so i just wanted to return something. picked 1.0.0